spec/fixtures/users.yml

```diff
@@ -0,0 +1,2 @@
+admin:
+  email: admin@example.com
```

Comment:
Fixture are the Rails default way of defining a default set of data in the database, to be used in tests.

Most projects replace fixtures with [FactoryBot](https://github.com/thoughtbot/factory_bot). We may go that way later, but using fixtures will be fine for now I think.